client/components/PhotoEntry.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const PhotoEntry = props => (
+  <div className="photo-tile">
+    <img src={props.link}></img>
+  </div>
+);
+
+export default PhotoEntry;
```

Comment:
Everything looks good here, maybe consider renaming the component to PhotoItem instead of PhotoEntry. The photo is not really an entry.